### DISCUSSION TITLE
[Question] Is there a way to customize the embed being shown on Discord?

### DISCUSSION THREAD

**Discussion by seriaati** [flet-dev/flet#4002] [Q&A] [answered]
### Question

When I post a link to my Flet application on Discord, it shows an embed like this:
![image](https://github.com/user-attachments/assets/f153ad55-7741-4bec-b84d-a124e00bec8c)

I would like to know if there is a way to customize this embed.

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment, marked as answer]
Docs: https://flet.dev/docs/publish/web/dynamic-website#assets
After packaging, open the `index.html` file and make the corresponding modifications.
